packages/terra-table/src/subcomponents/HeaderCheckMarkCell.module.scss

```diff
@@ -118,12 +118,12 @@
     background-color: transparent;
     background-image: none;
     border: 0;
-    height: 1;
     left: 50%;
-    margin: -1;
+    margin: -1px;
     padding: 0;
+    position: absolute;
     top: 50%;
-    width: 1;
+    width: 1px;
```

Comment:
Should this change have been reflected in the wdio screenshots?

Reply:
Looks like there was no wdio test written for the page (Multi Demo test) using this style. I have added on for the same bec551e.